guides/common/assembly_using-puppet-for-configuration-management.adoc

```diff
@@ -11,9 +11,16 @@ include::modules/con_puppet-architecture.adoc[leveloffset=+2]
 
 include::modules/proc_prerequisites-for-puppet.adoc[leveloffset=+2]
 
+ifdef::katello,satellite,orcharhino[]
+include::modules/proc_enabling-puppet.adoc[leveloffset=+2]
+endif::[]
+
```

Comment:
@ekohl This ifdef ensures that the whole procedure shows up only for Katello and its derivatives. Is that what you meant?

Reply:
No, I meant that there are katello-only options in the command. I'll expand a comment on https://github.com/theforeman/foreman-documentation/pull/890/files#r763216790 and others with background info.

Reply:
Actually, now that I'm reading it, perhaps you're right. In vanilla Foreman Puppet is enabled by default so this whole procedure is not needed.